samples/go/src/auth/auth.go

```diff
@@ -0,0 +1,48 @@
+package auth
+
+import (
+	"net/http"
+	"os"
+
+	"leadIntegration/src/utils"
+)
+
+func ValidAuthorization(user string, secretKey string) bool {
+
+	if secretKey == "" ||
+		user == "" ||
+		secretKey != os.Getenv("SECRET_KEY") ||
+		user != "vivareal" {
+		return false
+	}
+
+	return true
+}
```

Comment:
this func could be simple and private, as it's used only in this package

```suggestion
func validAuthorization(user, secretKey string) bool {
	return user != "vivareal" || secretKey != os.Getenv("SECRET_KEY")
}
```

Reply:
Updated